mm2src/common/transport/wasm_ws.rs

```diff
@@ -59,17 +134,19 @@ pub enum OutgoingError {
 }
 
 // TODO change the error type
-pub fn spawn_ws_transport(idx: ConnIdx, url: &str) -> Result<(WsOutgoingSender, WsIncomingReceiver), String> {
+pub fn spawn_ws_transport(idx: ConnIdx, url: &str) -> Result<(WsOutgoingSender, WsEventReceiver), String> {
```

Comment:
TODO above

Reply:
This TODO doesn't affect anything except the length of the debug entries. I was planning to figure out how to extract an error description without stack trace from the [JsValue](https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.WebSocket.html#method.new) on the next iteration.